### PR TITLE
Tests/integration/admin bike controller

### PR DESCRIPTION
**Pull request by BartoliniAndBorderCollies**
#### Issue
https://trello.com/c/e1K9UGIC

### Description
This branch adds integration tests for `AdminBikeController`

### Motivation
`Integration tests` side by side with `unit tests` are crucial elements of each application.

### Changes introduced by this pull request
- add `webflux` dependency
- add `mysql` dependency
- add integration tests
- refactor of fields in model classes to have the same names of fields (for model mapper)
- add `properties` to create a `separate database` for tests

### Testing
- [x] all tests passed
- [x] 100%  coverage of `AdminBikeController`

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -52,6 +52,15 @@
...
 		</dependency>
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-webflux</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>mysql</groupId>
+			<artifactId>mysql-connector-java</artifactId>
+			<version>8.0.33</version>
+		</dependency>
```
Comment (JasinskiSz) [inline pom.xml:63]: I think this dependency isn't required in this project due to the fact that you have spring-boot-starter-data-jpa dependency. But let me know if I'm wrong :)

**Review by JasinskiSz — COMMENTED** [review]
I've checked here all of the tests and they seem fine. Good job on that!

Let me know on my comment regarding dependencies and we'll continue.

File: pom.xml (hunk 1 above)
Comment (BartoliniAndBorderCollies) [reply pom.xml:63]: I want to have MySQL database for the test part of my application, therefore I need two dependencies. 
1. `The spring-boot-starter-data-jpa` is a starter for using Spring Data JPA with Hibernate. It simplifies the integration of Spring JPA/Hibernate with my project
2. `mysql-connector-java` is a JDBC driver that provides database connectivity through JDBC. This is needed for communication with MySQL db.

So I need both, `the spring-boot-starter-data-jpa` dependency for simplifying database interaction, and the `mysql-connector-java` dependency for connecting application to my MySQL database

Comment (JasinskiSz) [reply pom.xml:63]: But have you tried to run the application without the connector dependency? It looks like it works on my end without it.

Comment (BartoliniAndBorderCollies) [reply pom.xml:63]: I think it works without this dependency. We have to check it on our classes, cause I have one more issue with that! :)

Comment (BartoliniAndBorderCollies) [reply pom.xml:63]: OK, so it turns out this dependency is needed :)

**Review by JasinskiSz — APPROVED** [review]

**Pull request merged**